File: Arcade/TheCore/Javascript/leastFactorial.js
function fact(n) {
	if (n == 1) return n;
	return n * fact(n - 1);
}

function leastFactorial(n) {
    for (let i = 1; true; i++) {
        let f = fact(i)
        if (f >= n) return f;
    }
}

File: Arcade/TheCore/Java/LeastFactorial.java
final class LeastFactorial {
	
	int fact(int n) {
	    return n == 1 ? n : n * fact(n - 1);
	}

	int leastFactorial(int n) {
	    for (int i = 1; true; i++) {
	        int f = fact(i);
	        if (f >= n) return f;
	    }
	}
}